cloud/firestore/v1/document_event_data.go

```diff
@@ -11,18 +11,17 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-package firestore
 
-import "encoding/json"
+package firestore
 
-// The data within all Firestore document events.
+// DocumentEventData: The data within all Firestore document events.
```

Comment:
nit: This is more commonly used in a sentence form, such as:

```suggestion
// DocumentEventData type represents the data in a Firestore document event.
```

Reply:
OK.

I don't know how to modify the proto descriptions to conform to the style of every programming language comment form. This change to the post-proto descriptions seemed like a reasonable compromise.